### ISSUE TITLE
Bug: Driver failed to activate with new version 0.3.2 (beta): still problems

### ISSUE THREAD

**Issue by erwinxxx** [bitgapp/eqMac#330]
Describe the bug
Still llot's of problems with the new version of EqMac 0.3.2 (beta) on my Mac Mini 10.15.5. It try uninstall it with the sudo comand for it, and reinstall it but still "Driver failed to activate" error after that. Try again doesn't help.
But after a reboot of my Mac, the Install audio driver message came up and install it and it works at the moment. But after I install the latest version yesterday I did the same but this morning the Driver failed to activate came up again.
After a couple of days of good working of the EqMac app a couple of days ago again the 'Driver Failed To Activate' error after a reboot of my Mac. I reinstall it, reboot my Mac and it's working again. And also this morning the same isssue and now I totally deinstall the EqMac app and fully install it again, the first time after the reboot I get the same message and after another reboot I get the message to reinstall the audio driver and now it's working for the time being again, I guess :-(.

And another strange thing. I deinstall it but my preset I made was still there so it not deinstall all the files. When I check this with the app 'find any file' there stil some eqmac files left behind after a deinstall. So that's not good.

Steps to Reproduce
Steps to reproduce the behaviour (feel free to change the placeholder as you need):

    Startup my Mac
    EqMac show up
    Driver failed to activate
    full deinstall it, reinstall it but again 'driver failed to activate again. After another reboot I get the install audio driver and it's
   working normaly

Expected behaviour

Setup information:

    Audio device used for playback: [eg AirPods or Macbook speaker or Bose QC32]
    Audio transmission interface: [eq HDMI or Bluetooth or 3.5mm Jack]
    macOS Version: [e.g. 10.15]
    eqMac Version [e.g. 0.3.2] (if audio related issue)
    eqMac UI Version [e.g. ]1.2.1 (if UI related issue)
    Driver version: 1.1.1

Screenshots or Console.app logs
If applicable, add screenshots to help explain your problem. Also you can run Console.app and apply filter for eqMac

Additional information

**Comment by erwinxxx** [comment]
Still got the same issues over and over again after a couple days of working properly. Hope the developer can look into this a.s.a.p. this is really annoying.

**Comment by nivshah** [comment]
I am having this issue as well. I have been tracking #225 and noticed the recommendation to get 0.3.2. I tried clicking "Check for update" in app (0.3.1) and didn't see anything, so I downloaded the 0.3.2 release and overwrote the existing eqmac installation. This may not be the correct upgrade path.

Installing the audio driver update failed, so I restarted my computer. The audio driver installed after a restart but now when I run eqmac it gives an error saying the driver is not activating. I've uninstalled eqmac (dragged to trash, emptied trash) and reinstalled with no luck - I get "Driver failed to activate" every time I try to start eqmac 0.3.2 now.

**Comment by hlvrn** [comment]
I'm having similar problems. I first installed EqMac (latest version 0.3.2), it worked. Then I installed EqMac2, and it asked to uninstall old drivers, I clicked ok (which I now regret). EqMac2 didn't work, so I uninstalled it, and I was hoping that EqMac would work like before, but it doesn't. Now, every time I open EqMac it says "Unfortunately the audio driver has failed to active. You can restart eqMac and try again or quit."

**Comment by erwinxxx** [comment]
Another issue. After a week or so no Driver Failed to activate messages anymore so far. But sometimes after a reboot EqMac is not starting up. I only see that white box but EqMac is not starting. After another reboot it's working again. Still very buggy.

**Comment by RonProchot** [comment]
Hello. I reinstalled the most recent (beta) version of eqMac (v0.3.2) on top of the existing application in my system, which ran SUPERBLY well last three days ago (the first and only time I've actually used it), but which would not load today due to the "Drive failed message" indicated above. However the new installation resulted in an system error message: "Problem Report for eqMac - eqMac quite unexpectedly", with the BIG exclamation mark.  This may have had something to do with Apple's most recent update last night, to OS 10.15.6, on my 2014 iMac Retina, 5K, QuadCore with 32GB, 1600MHz, DDR3 RAM. These reports make no sense to me, other than it does state: 
Exception Type: EXC_BAD_ACCESS (SIGSEGV)
Exception Codes: KERN_INVALID_ADDRESS at 0x0000000000000000
Exception Note:  EXC_CORPSE_NOTIFY  
Plus a whole bunch of other stuff, too much to post here. Does anyone have any suggestions on what I might do next to get the application working. It sounds fantastic when it works.  Thanks!

**Comment by RonProchot** [comment]
Hi. I have successfully installed eqMac v0.3.2, and after rebooting managed to get it to work. So far, so good!  I removed the previous version of eqMac v0.3.2 from the Applications folder with a right click and emptied the trash. Downloaded a new version through the OWC link at Rocket Yard and installed it (just as I did previously)as per the set-up instructions. Following the install I did reboot and then opened the application, from the Applications folder. To actually see the eqMac equalizer window, I had to click on the eqMac icon in the Doc. In case you're interested my System is an iMac 27" Retina Display, Late 2014 (as above) with Mac OSX 10.15.6 and eqMac v0.3.2. Thanks all for your help, in particular the people at Rocket Yard, and of course the developer of this nifty program. Stay safe out there and remember to practice Safe Social Distancing.

**Comment by trbofr13nd** [comment]
Having this problem too. I just started my computer and it didn't turn on with it, and is telling me the driver failed to activate. Tried reinstalling and no luck! Can't do anything else within the app but retry and quit.

**Comment by Proper-Job** [comment]
As a workaround I quit eqMac, executed this from the Terminal, reopend eqMac and allowed it to reinstall the driver. I got (beautiful) sound after that :)
```shell
sudo /Applications/eqMac.app/Contents/Resources/uninstall_driver.sh
```

**Comment by RonProchot** [comment]
Thanks Moritz.

**Comment by Kosuke-Ito** [comment]
@Proper-Job
It's all taken care of. Thank you.

**Comment by nodeful** [comment]
Thanks for being patient with me.
I have just released v1.0.0. You can download it here: https://github.com/bitgapp/eqMac/releases/tag/v1.0.0
Or from the website: https://eqmac.app
Could you please let me know if this issue is still present. 
Thanks again.

Roman

**Comment by gitstuffedm8** [comment]
> 
> 
> Thanks for being patient with me.
> I have just released v1.0.0. You can download it here: https://github.com/bitgapp/eqMac/releases/tag/v1.0.0
> Or from the website: https://eqmac.app
> Could you please let me know if this issue is still present.
> Thanks again.
> 
> Roman

Hi Roman, the issue is resolved for me. Same problem was here - installed eqmac today and then audio driver error came up same as people above. Funnily enough the download was from eqmac.app so should be latest version? Anyway I restart coreaudio, reboot a few times, nothing helps. Then I uninstalled, downloaded version 1.0.0 from github from your link above, then installed, and it worked. Didn't even have to reboot.

Thank you for resolving and for your effort put into this app!

**Comment by nodeful** [comment]
I think this issue has been resolved for a while now, so I'll be closing this issue unless suggested otherwise. Thank you
